core-backend/src/funcs.rs

```diff
@@ -675,6 +675,26 @@ where
         })
     }
 
+    pub fn free_range(start: u32, end: u32) -> impl SysCall<Ext> {
```

Comment:
I think we need to discuss sys-call error design with @breathx & @grishasobol

Reply:
It has been already designed:
1) Nothing is returned to user
2) Error occurs only if `end` is out of memory bounds or `start` > `end`

Reply:
> Nothing is returned to user

Why?

Reply:
This is the most wide and performance efficient behaviour

Reply:
About returning error to user - what the reason? Some body will check that start < end using free range?

Reply:
added error, we no longer trap if range is out of bounds